CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.8)
+cmake_minimum_required (VERSION 3.0)
```

Comment:
> The warnings you get are likely only because of problems within Qt-CMake---nothing we should worry about. We do not use `COMPILE_DEFINITIONS_*`. Workaround: set the policy within `qt-gui`.

Oh, I guess I will remove the version bump then once again ☺️.